src/main/java/mate/academy/intro/dto/book/BookSearchParametersDto.java

```diff
@@ -1,4 +1,4 @@
 package mate.academy.intro.dto.book;
 
-public record BookSearchParametersDto(String[] titles, String[] authors) {
+public record BookSearchParametersDto(String[] titles, String[] authors, String[] price) {
```

Comment:
how do you imagine this price array? How long?
{100.01, 100.02, 100.03, ...... 158.52, 158.53 ......}